src/views/puzzles/PuzzleView/PuzzleView.vue

```diff
@@ -44,7 +44,9 @@
         </template>
         <ul style="padding: 0; list-style-type:none" v-if="puzzle">
           <li v-if="madeByPlayer">
-            <img :src="avatar" alt="author" class="icon" />{{ puzzle.username }}
+            <router-link :to="`/players/${puzzle.uid}`">
+              <img :src="avatar" alt="author" class="icon" />{{ puzzle.username }}
```

Comment:
Minor: Should the image be included in the link? Visually we're treating the image as a label like other icons - I'm not sure that it would be expected to be able to click the image to get to the player profile?

Reply:
You make an interesting point. On this very Github PR page, both approaches are used.

Reply:
Wow didn't notice that inconsistency before - yeah not sure what the right answer is